src/entrypoints.php

```diff
@@ -41,7 +41,7 @@ function json_from_file(string $file): dict<string, mixed> {
     /* depth = */ 512 /* == default */,
     \JSON_FB_HACK_ARRAYS,
   );
-  if (!is_dict($no_type_refinment_please = $json)) {
+  if (!is_dict(/* $no_type_refinment_please = */ $json)) {
```

Comment:
this introduces a type error - possibly a case for `/* HHAST_IGNORE_ERROR[YourLinterNameHere] */`, or safe it to a local before this